opal/io/wait.rb

```diff
@@ -0,0 +1,34 @@
+class ::IO
```

Comment:
Maybe it would make more sense to move those into the `stdlib`?

Reply:
These are tested in spec/ruby/core so i would prefer to have them in corelib. They don't have a real implementation yet, i add it to todo